.github/actions/test-khiops-on-iris/action.yml

```diff
@@ -0,0 +1,74 @@
+---
+name: Test Khiops on Iris
+description: Tests a Khiops installation on the Iris dataset
+runs:
+  using: composite
+  steps:
+    # Python install: we don't use the setup-python action because of the following error:
+    # python3: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by python3)
+    - name: Python setup for Linux
+      if: runner.os == 'Linux'
+      shell: bash
+      run: |
+        if [ -d "/etc/apt" ]
```

Comment:
Is  this a robust enough way of sorting out the Ubuntu / Rocky dichotomy? What would prevent some user from creating the `/etc/apt` directory on a Rocky OS?

I would have relied on the relevant env vars `/etc/os-release` instead, for this.

Reply:
You're right, on standard environment, it is not enough. But we're in a controlled environment.

Reply:
Ok, then could you add a comment before the `if`, something like `# Debian-based OS:`?

Reply:
done